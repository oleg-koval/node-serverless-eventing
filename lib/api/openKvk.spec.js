const { getApiParams } = require('./openKvk')

describe('getApiParams', () => {
    it('should collect all params into one object', () => {
        const res = getApiParams('42')
        expect(res).toMatchSnapshot()
    })
})
