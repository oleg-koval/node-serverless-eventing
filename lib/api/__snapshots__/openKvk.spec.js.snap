// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getApiParams should collect all params into one object 1`] = `
Object {
  "headers": Object {
    "ovio-api-key": "secret",
  },
  "json": true,
  "uri": "https://api.overheid.io/openkvk/42",
}
`;
